public/app/percona/dbaas/components/DBCluster/EditDBClusterModal/EditDBClusterModal.tsx

```diff
@@ -40,10 +39,10 @@ export const EditDBClusterModal: FC<EditDBClusterModalProps> = ({
         databaseType: selectedCluster.databaseType,
         clusterName: selectedCluster.clusterName,
         kubernetesClusterName: selectedCluster.kubernetesClusterName,
-        clusterSize: topology === DBClusterTopology.cluster ? nodes : single,
-        cpu,
-        memory,
-        disk,
+        clusterSize: topology === DBClusterTopology.cluster ? +nodes : +single,
```

Comment:
Is the `+` here because of the linter (that it expects a number)?

Reply:
linter asked to remove any, but we have both string and number type there, i can disable it or do this way. Do you think it's better just disable this line?

Reply:
i removed this and tried vs code, it works there)